subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1821,11 +1841,15 @@ static int wifi_ap_config_args_to_params(const struct shell *sh, size_t argc, ch
 	static const struct option long_options[] = {
 		{"max_inactivity", required_argument, 0, 'i'},
 		{"max_num_sta", required_argument, 0, 's'},
+#if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT)
+		{"ht_capab", required_argument, 0, 'n'},
+		{"vht_capab", required_argument, 0, 'c'},
```

Comment:
`htc` and `vhtc`  as short forms?

Reply:
Sorry, which one?  
If you mean `ht_capab` -> `htc`,  here `ht_capab` is to be consistent with the hostapd.conf file. If the user wants to set it, they can directly refer to the hostapd.conf file.
If you mean `n`->`htc`, strings shouldn't be used here, right?

Reply:
> If you mean n->htc, strings shouldn't be used here, right?

I was referring to this, if you use `getopt_long` then you can use multicharacter short options, but anyways not mandatory.